tests/main/lxd/task.yaml

```diff
@@ -17,13 +17,28 @@ kill-timeout: 25m
 # Start before anything else as it can take a really long time.
 priority: 1000
 
+environment:
+    REFRESH_APP_AWARENESS_OUTER/snapd_cgroup_just_inside: false
+    REFRESH_APP_AWARENESS_INNER/snapd_cgroup_just_inside: true
+    REFRESH_APP_AWARENESS_OUTER/snapd_cgroup_just_outside: true
+    REFRESH_APP_AWARENESS_INNER/snapd_cgroup_just_outside: false
+    REFRESH_APP_AWARENESS_OUTER/snapd_cgroup_both: true
+    REFRESH_APP_AWARENESS_INNER/snapd_cgroup_both: true
+    REFRESH_APP_AWARENESS_OUTER/snapd_cgroup_neither: false
+    REFRESH_APP_AWARENESS_INNER/snapd_cgroup_neither: false
+    LXD_CHANNEL: candidate
```

Comment:
this is already available globally via spread.yaml's `LXD_SNAP_CHANNEL`, I would prefer to use this so we don't have to remember which tests use which channel and can easily roll back to a different track if latest gets broken for some time (as has happened a few times)

Reply:
Thanks for spotting this! It was a dead variable, we actually already use `LXD_SNAP_CHANNEL` in the test below. The change predates this by a lot as it was made in Nov 2019.